src/librustc_allocator/expand.rs

```diff
@@ -28,13 +29,14 @@ use syntax::{
     fold::{self, Folder},
     parse::ParseSess,
     ptr::P,
-    symbol::Symbol,
-    util::small_vector::SmallVector,
+    symbol::Symbol
 };
 use syntax_pos::Span;
 
 use {AllocatorMethod, AllocatorTy, ALLOCATOR_METHODS};
 
+type SmallVector<T> = SmallVec<[T; 1]>;
```

Comment:
Let's have a single definition of this, probably in data structures.

Reply:
I thought about this, but it might reduce the tweakability of `SmallVec` for different modules; are we sure this size fits all?

Reply:
Not all modules will use it; `SmallVec` will still be exported. Ideally we'd rename `SmallVector` to `OneVector`.

Reply:
Ok; I'll update and rebase soon.